deepfence_frontend/apps/dashboard/src/features/onboard/components/connectors/hosts/DockerConnectorForm.tsx

```diff
@@ -0,0 +1,73 @@
+import cx from 'classnames';
+import { HiViewGridAdd } from 'react-icons/hi';
+import { Button, Card, Step, Stepper, Typography } from 'ui-components';
+
+import { CopyToClipboardIcon } from '../../../../../components/CopyToClipboardIcon';
+import { usePageNavigation } from '../../../../../utils/usePageNavigation';
+
+export const DockerConnectorForm = () => {
+  const { navigate } = usePageNavigation();
+
+  const code = `docker run -dit --cpus=".2" --name=deepfence-agent --restart on-failure --pid=host --net=host \\
+  --privileged=true -v /sys/kernel/debug:/sys/kernel/debug:rw -v /var/log/fenced \\
+  -v /var/run/docker.sock:/var/run/docker.sock -v /:/fenced/mnt/host/:ro \\
+  -e USER_DEFINED_TAGS="" -e MGMT_CONSOLE_URL="${
+    window.location.host ?? '---CONSOLE-IP---'
+  }" -e MGMT_CONSOLE_PORT="443" \\
+  -e DEEPFENCE_KEY="${localStorage.getItem('dfApiKey') ?? '---DEEPFENCE-API-KEY---'}" \\
+  deepfenceio/deepfence_agent_ce:${''}`;
```

Comment:
use `latest` as tag for now, later we will detect the console version and use it here.

Reply:
Okay